src/Miningcore/Blockchain/Ethereum/EthereumJobManager.cs

```diff
@@ -352,9 +356,24 @@ public override void Configure(PoolConfig pc, ClusterConfig cc)
 
             // create it if necessary
             Directory.CreateDirectory(dagDir);
+            
```

Comment:
```c#
 var dagDir = !string.IsNullOrEmpty(extraPoolConfig?.DagDir) ?
                Environment.ExpandEnvironmentVariables(extraPoolConfig.DagDir) :
                Dag.GetDefaultDagDirectory();
```

should probably be handled on a per "algo" basis, since each algo has it's own `Dag.GetDefaultDagDirectory()` function.

Reply:
Yes absolutely. I can believe i actually missed it. i will do it right away and test it.